AndroidSDK/src/com/leanplum/messagetemplates/MessageTemplates.java

```diff
@@ -91,17 +92,17 @@
     static final String DEFAULT_URL = "http://www.example.com";
 
     // Web interstitial values.
-    static final String DEFAULT_CLOSE_URL = "http://leanplum:close";
+    static final String DEFAULT_CLOSE_URL = "https://leanplum/close";
     static final boolean DEFAULT_HAS_DISMISS_BUTTON = true;
 
     // HTML Template values.
     public static final String FILE_PREFIX = "__file__";
     public static final String HTML_TEMPLATE_PREFIX = "__file__Template";
-    static final String DEFAULT_OPEN_URL = "http://leanplum:loadFinished";
-    static final String DEFAULT_TRACK_URL = "http://leanplum:track";
-    static final String DEFAULT_ACTION_URL = "http://leanplum:runAction";
-    static final String DEFAULT_TRACK_ACTION_URL = "http://leanplum:runTrackedAction";
-
+    static final String DEFAULT_OPEN_URL = "https://leanplum/loadFinished";
```

Comment:
I think it would be safer to handle both the old and new URLs. It shouldn't be possible but just in case they got the new SDK with an old template we should cover that case.